doc/zmq_close.txt

```diff
@@ -33,9 +37,9 @@ return `-1` and set 'errno' to one of the values defined below.
 
 
 ERRORS
-------
+-------
```

Comment:
Shouldn't this line match exactly the one above? Otherwise the web doc formatting will break

Reply:
Yes, fixed that